custom_model_runner/datarobot_drum/drum/args_parser.py

```diff
@@ -946,6 +946,8 @@ def verify_monitoring_options(options, parser_name):
                     missing_args.append(ArgumentsOptions.DR_WEBSERVER)
                 if options.api_token is None:
                     missing_args.append(ArgumentsOptions.DR_API_TOKEN)
+                if options.monitor_settings is None:
```

Comment:
Will it make sense to support default monitor settings? With that, we'll avoid compatibility issue with DataRobot.

Reply:
If monitor settings are not provided, we consider default spool channel to be filesystem